juan_vides/ejercicio2.md

```diff
@@ -0,0 +1,20 @@
+```
+Sub ejercicio()
+    fila = 2
+    sw = True
+    cedula = Int(nume.Cells(9, 4))
```

Comment:
Can you give me details about this?

Reply:
Hello, of course I can explain, I use 2 sheets for my code because in one is the executor where the user enters the number and in the same excel gives an answer, in the other is the data record with the information; I hope I have explained.

Reply:
Alright. I understand.